tsconfig.json

```diff
@@ -1,19 +1,11 @@
 {
+    "extends": "tsconfigs/browser-module",
     "compilerOptions": {
+        "esModuleInterop": false,
+        "strictFunctionTypes": false,
         "target": "ES5",
-        "noImplicitAny": true,
-        "sourceMap": true,
-        "strictNullChecks": true,
-        "declaration": true,
-        "removeComments": false,
```

Comment:
[TypeScript default is `false`](https://www.typescriptlang.org/docs/handbook/compiler-options.html) and tsconfigs doesn't set it.